tests/broker-engine/tags.robot

```diff
@@ -705,6 +705,101 @@ BEUTAG_REMOVE_HOST_FROM_HOSTGROUP
     Should Be True    ${result}    Host 3 should have hostgroup tags 2
 
 
+MOVE_HOST_OF_HOSTGROUP_TO_ANOTHER_POLLER
+    [Documentation]    Given two pollers with two hosts on each of them that belong to the same hostgroup, when I move two hosts from one poller to another, then the hostgroup tag is not erased.
+    [Tags]    broker    engine    tags    MON-146720
+    Ctn Clear Db    tags
+    Ctn Clear Db    resources
+    Ctn Clear Db    resources_tags
+
+    Ctn Config Engine    ${2}    ${8}    ${5}
+    Ctn Create Tags File    ${0}    ${1}    ${0}    hostgroup
+    Ctn Create Tags File    ${1}    ${1}    ${0}    hostgroup
+    Ctn Config Engine Add Cfg File    ${0}    tags.cfg
+    Ctn Config Engine Add Cfg File    ${1}    tags.cfg
+    Ctn Add Tags To Hosts    ${0}    group_tags    1    ['host_1', 'host_2']
+    Ctn Add Tags To Hosts    ${1}    group_tags    1    ['host_5', 'host_6']
+    Ctn Add Host Group    ${0}    1    ['host_1', 'host_2']
+    Ctn Add Host Group    ${1}    1    ['host_5', 'host_6']
+    Ctn Config Broker    central
+    Ctn Config Broker    rrd
+    Ctn Config Broker    module    ${2}
+    Ctn Config Broker Sql Output    central    unified_sql
+    Ctn Config BBDO3    ${2}
+    Ctn Broker Config Log    module0    neb    debug
+    Ctn Broker Config Log    central    sql    trace
+    Ctn Clear Retention
+
+    Sleep    1s
+    ${start}    Get Current Date
+    Ctn Start engine
+    Ctn Start Broker
+
+    ${result}    Ctn Check Resources Tags With Timeout    0    1    hostgroup    [1]    60    True
+    Should Be True    ${result}    Host 1 should have hostgroup tags 1
+    ${result}    Ctn Check Resources Tags With Timeout    0    2    hostgroup    [1]    60    True
+    Should Be True    ${result}    Host 2 should have hostgroup tags 1
+    ${result}    Ctn Check Resources Tags With Timeout    0    5    hostgroup    [1]    60    True
+    Should Be True    ${result}    Host 5 should have hostgroup tags 1
+    ${result}    Ctn Check Resources Tags With Timeout    0    6    hostgroup    [1]    60    True
+    Should Be True    ${result}    Host 6 should have hostgroup tags 1
+
+    #remove host_5 and host_6 from poller 1
+    Log To Console    Remove host_5 and host_6 from poller 1
+    Ctn Engine Config Remove Host    ${1}    host_5
+    Ctn Engine Config Remove Host    ${1}    host_6
+    Ctn Engine Config Remove Service Host    ${1}    host_5
+    Ctn Engine Config Remove Service Host    ${1}    host_6
+    Ctn Engine Config Remove Tag    ${1}    1
+
+    ${start}    Get Current Date
+    Ctn Reload Engine    poller_index=${1}
+    Ctn Reload Broker
+
+    Sleep    5s
```

Comment:
Why this tempo? Just after we wait for something in the database for 60s. If these 5s are really needed, why not wait for 65s in the next step?